test/integration-tests/test-utilities.js

```diff
@@ -211,11 +270,12 @@ async function createResources(folderName, url, randString, enabled) {
  * Wraps deletion of testing resources.
  * @param folderName
  * @param url
+ * @param randString
  * @returns {Promise<boolean>} Resolves true if resources destroyed, else false.
  */
-async function destroyResources(folderName, url) {
+async function destroyResources(folderName, url, randString) {
```

Comment:
`randString` could have a better name to explain what this is being used for. Like domainSuffix or domainPrefix